src/index.js

```diff
@@ -17,6 +17,14 @@ const router = new koaRouter()
 createAPIRoutes(router)
 createMiddlewareRoute(router)
 
+router.get('/_health', (ctx, next) => {
```

Comment:
Hey, is there a reason for having the underscore in the path? Its a bit confusing

Reply:
I've seen this underscore prefix used for health check paths before. 
There are a few reasons I used the underscore:
- The underscore prefix in JS indicates to developers that this concept is "internal". This Healthcheck is not meant for your typical user - this is meant for high-level users to check our API is accessible
- On that same reasoning, because underscores are seen as "internal" users wouldn't normally add prefixing underscores to their custom patterns. Therefore, by adding the underscore we aren't using up a URL pattern that our users might want ie: `health`

I can't find any articles to properly support me though - so I'm happy to change it to something better. @BMartinos ?

Reply:
I think if this is only going to be used internally then a prefixed underscore seems fine to me. Prefixed underscores are generally for internal use or when the variable in the function inst used.

I do think this endpoint would have value externally through if other services want to check if the mediator is running so updating it to `/heartbeat` might be better

Reply:
cool, I thought that only applied to variables and functions